terraform/main.tf

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+# https://www.terraform.io/docs/providers/aws/
+provider "aws" {
+  alias  = "configured"
```

Comment:
You don't really need this unless you are going to reference the "configured" part somewhere